src/services/filter/filters/evm/evaluator.rs

```diff
@@ -726,119 +676,6 @@ impl<'a> EVMConditionEvaluator<'a> {
 			}
 		}
 	}
-
-	/// Compares a map (JSON object) value with a literal value.
-	///
-	/// Arguments:
-	/// - lhs_json_map_str: The left-hand side value as a JSON map string.
-	/// - operator: The operator to use for the comparison.
-	/// - rhs_literal: The right-hand side value.
-	///
-	/// Returns:
-	/// - true if the comparison is true, false otherwise.
-	/// - error if the comparison is not supported.
-	pub fn compare_map(
```

Comment:
I wonder, why are we removing map? Is this because in solidity, maps/structs are casted to tuples? cc @NicoMolinaOZ